### DISCUSSION TITLE
CDT Call January 2025

### DISCUSSION THREAD

**Discussion by jonahgraham** [eclipse-cdt/cdt#967] [Announcements]
Our next CDT Call will be January 8th at 11am Ottawa time. Please make a comment to add suggestions for discussion topics, or come along to the call and we can discuss anything in "Any other business" section. Everyone is welcome.

## Call Information

<details><summary>Call Information and Agenda</summary>
<p>

- **Location:** [:computer: Zoom Meeting](https://eclipse.zoom.us/j/545364607?pwd=L1d2eDQ4RWJmdkk3YXkvcW90RWJiQT09) ([full details below](#Connection-Details))
- **Date:** January 8, 2024 11:00 AM Eastern Time (US and Canada) ([Calendar/ICS](https://eclipse.zoom.us/meeting/uJAodO-sqDstovAF50NQR9k2f8kNjAOvEg/ics?icsToken=98tyKuysqzgsHtKVt1z9c6ktOcH-b9_yln5Yi4UIzDHUMXlYMAWuPcxIE51pNd-B)) ([world clock](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2025&month=01&day=08&hour=16&min=0&sec=0&p1=188&p2=136&p3=195))
- **Agenda**
1. Welcome
2. Actions from last meeting
3. Initial discussion on https://github.com/eclipse-cdt/cdt/discussions/988 - follow-up next meeting when Tobias can attend
4. https://github.com/eclipse-cdt/cdt/discussions/949
5. https://github.com/eclipse-cdt/cdt/pull/960#issuecomment-2575221657
6. https://github.com/eclipse-cdt/cdt/pull/995#issuecomment-2577852055
8. CDT 12.0 planning
9. IDE use case demonstration from Erwin
10. Any other business?
- **Contact:** <cdt-dev@eclipse.org> or [start a discussion topic](https://github.com/eclipse-cdt/cdt/discussions)
- **Reference:** [Previous and future meetings](https://github.com/eclipse-cdt/cdt/discussions?discussions_q=label%3Acdtcalls) (older call notes are located [here](https://hackmd.io/@jonahgraham/EclipseCDTCalls))

## Connection Details

Join Zoom Meeting
https://eclipse.zoom.us/j/545364607
Meeting ID: 545 364 607
Passcode: 196590
Dial by your location find your local number: https://eclipse.zoom.us/u/acMY3kyoJW

</p>
</details> 

## Minutes

- Initial discussion on https://github.com/eclipse-cdt/cdt/discussions/988 - follow-up next meeting when Tobias can attend
  - Can be added to https://github.com/eclipse-cdt/cdt?tab=readme-ov-file#add-ons-for-cdt
  - Very welcome to the OSS community!

- https://github.com/eclipse-cdt/cdt/discussions/949
  - supporting Rust sounds like a good idea, when we get an owner to support ongoing Rust work we should revisit again

- https://github.com/eclipse-cdt/cdt/pull/960#issuecomment-2575221657
  - progressed in the call - view the review comments and discussions in #960 for more info
  - asciidoc is very slow on some machines to render, perhaps because of fontawesome. Consider https://docs.asciidoctor.org/asciidoctor/latest/html-backend/local-font-awesome/ or dropping fontawesome entirely - @jonahgraham will handle this as part of #995 
  - this will need to be rebased to integrate into the refactoring done in #995

- https://github.com/eclipse-cdt/cdt/pull/995#issuecomment-2577852055
  - don't include generated html
  - try to have pipeline publish generated html for easy review

- CDT 12.0 planning
  - Jan 27 is deadline for most major changes

- IDE use case demonstration from Erwin
  - Great demo from Erwin that gave us all a good insight how CDT can be used and the driving forces behind why he has contributed the improvements to CDT that have been added recently.

- Next call February 12
  - https://github.com/eclipse-cdt/cdt/discussions/996

**Comment by jonahgraham** [comment]
Additional agenda item:

- https://github.com/eclipse-cdt/cdt/discussions/988

**Reply by jonahgraham** [reply]
- Comment on  #960 https://github.com/eclipse-cdt/cdt/pull/960/files/80e1aaebd16ea3c3926e55bc5c1caf7d63502924#r1882260989

**Reply by jonahgraham** [reply]
- https://github.com/eclipse-cdt/cdt/discussions/949

**Reply by jonahgraham** [reply]
- https://github.com/eclipse-cdt/cdt/pull/960#issuecomment-2575221657

**Reply by thahnen** [reply]
I'm sorry, but due to an unforeseen sickness I'll be sadly not able to attend today. I will definitely attend the next meeting! I hope it is okey with you to shift this agenda item to the next one.

In the meantime I'll live [THIS](https://github.com/SonarSource/sonarlint-eclipse/tree/master/org.sonarlint.eclipse.cdt) here at least, this is our CDT connectior, we currently rely on this and not on the compilation database (yet).

Best,
Tobias

**Reply by jonahgraham** [reply]
* https://github.com/eclipse-cdt/cdt/pull/995#issuecomment-2577852055

**Reply by jonahgraham** [reply]
> I'm sorry, but due to an unforeseen sickness I'll be sadly not able to attend today. I will definitely attend the next meeting! I hope it is okey with you to shift this agenda item to the next one.

Get well soon! I look forward to seeing you on Feb 12. I'll be making the discussion entry for that meeting soon
